Tensile/Tests/nightly/assertions/test_hgemm_asem2_asm.yaml

```diff
@@ -23,6 +23,7 @@ BenchmarkProblems:
     - # ProblemType
       OperationType: GEMM
       DataType: h
+      DestDataType: h
```

Comment:
Would a default assignment work here (if DestDataType not defined then defaults to DataType) ?  Would be less disruptive in long run for existing YAML files.